jest.config.js

```diff
@@ -2,8 +2,10 @@
  * NOTE: hd-blocks mapper is needed to solve hd-blocks path in the components
  */
 module.exports = {
+  globalSetup: './tests/global-setup.js',
   collectCoverageFrom: [
     'src/components/**/*.vue',
+    'src/services/**/*.js',
```

Comment:
Adding services to coverage reporting.

Reply:
That obviously dropped our overall coverage. :) I'm not going to cover all the services with tests in this PR, so I might loosen up rules for merging to get this in. :)